proto/proto_test.go

```diff
@@ -153,10 +153,12 @@ func TestMarshalUnmarshal(t *testing.T) {
 
 		// float
 		float32(0),
+		float32(-0.0),
 		float32(0.1234),
 
 		// double
 		float64(0),
+		float64(-0.0),
```

Comment:
I don't this works; you need to use `math.Copysign(0, -1)`. See https://github.com/golang/go/issues/2196 and https://github.com/golang/go/commit/adba6c4fdf8c9d2078a88a016924e80fd23cb39c.

Reply:
Thanks 👍